blazor/pdfviewer/getting-started/server-side-application.md

```diff
@@ -1,46 +1,37 @@
 ---
 layout: post
 title: Server-side application in Blazor PDF Viewer Component | Syncfusion
```

Comment:
Change the title content like this "Getting Started with PDF Viewer in Blazor Server App | Syncfusion"

Reply:
For WASM, change the title as "Getting Started with PDF Viewer in Blazor WASM App | Syncfusion"

Reply:
Changed the title as per the suggestion